examples/cms/src/data.ts

```diff
@@ -0,0 +1,166 @@
+import { CollectionItemData, ManagedCollection, ManagedCollectionField, framer } from "framer-plugin"
+import { generateHash, isNotNull } from "./utils"
+import { PLUGIN_KEYS } from "./constants"
+
+export type ManagedCollectionFieldType = ManagedCollectionField["type"]
+
+type DataSourceField = {
+    type: ManagedCollectionFieldType
+}
+
+export interface DataSource {
+    id: string
+    fields: Record<string, DataSourceField>
+    items: Record<string, unknown>[]
+}
+
+export interface FieldConfig {
+    name: string
+    isNew: boolean
+    field: ManagedCollectionField | null
+}
+
+export function listDataSourcesIds(): string[] {
+    return ["articles", "categories"]
+}
+
+export async function getDataSources(collection: string): Promise<DataSource> {
+    return await fetch(`/datasources/${collection}.json`).then(response => response.json())
+}
+
+export function computeFieldConfigs(existingFields: ManagedCollectionField[], dataSource: DataSource) {
+    const result: FieldConfig[] = []
+    const fields = dataSource.fields
+
+    for (const [name, fieldType] of Object.entries(fields)) {
+        const fieldId = generateHash(name)
```

Comment:
What's the reason for using a hash here?

Reply:
There is no reason to keep the hash anymore, this can be removed everywhere now, before we used to hash values because we were storing them.

Reply:
Great. Let's clean it up 🧹